addon/views/sl-split-grid-row.js

```diff
@@ -0,0 +1,64 @@
+import Ember from 'ember';
+
+/**
+ * @module views
+ * @class  sl-split-grid-row
+ */
+export default Ember.View.extend( Ember.ViewTargetActionSupport, {
+
+    // -------------------------------------------------------------------------
+    // Dependencies
+
+    // -------------------------------------------------------------------------
+    // Attributes
+
+    /**
+     * The HTML tag name for the view element
+     *
+     * @property {string} tagName
```

Comment:
All of the other files in this repository use Ember.String. Please change the occurrence for this property as well as for all other similar instances in this file to be consistent with the rest of the codebase. If it is desired to move away from the "Ember." syntax create an issue to be worked on separately.